src/main/java/PrintUtil.java

```diff
@@ -0,0 +1,41 @@
+import java.io.PrintStream;
+
+/**
+ * Class containing static methods for printing formatted text to the UI.
+ */
+public class PrintUtil {
+    private static final int PRINT_INDENT_LEVELS = 4;
+    
+    private static String indentString(String s) {
+        //https://stackoverflow.com/questions/15888934/how-to-indent-a-multi-line-paragraph-being-written-to-the-console-in-java
+        return s.replaceAll("(?m)^", " ".repeat(PRINT_INDENT_LEVELS));
```

Comment:
I think it is good that you quoted the source of your code! Perhaps you could consider adding some comments to explain the part that you used from the code provided in the link; this will definitely make it clearer!